src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/UploadDocumentHelper.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.helpers;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.util.CollectionUtils;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.et.common.model.ccd.CaseDetails;
+import uk.gov.hmcts.et.common.model.ccd.items.DocumentTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.types.ClaimantIndType;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+
+@Slf4j
+public class UploadDocumentHelper {
+
+    private UploadDocumentHelper() {
+        // Access through static methods
+    }
+
+    /**
+     * Returns true if the case is rejected, has a rejection of claim document and has not got an email sent flag set.
+     * @param caseDetails details of the case
+     */
+    public static Boolean shouldSendRejectionEmail(CaseDetails caseDetails) {
+        CaseData caseData = caseDetails.getCaseData();
+        List<DocumentTypeItem> documentCollection = caseData.getDocumentCollection();
+
+        if (CollectionUtils.isEmpty(documentCollection)) {
+            return false;
+        }
+
+        Optional<DocumentTypeItem> rejectionDocument = documentCollection.stream().filter(o ->
+            "Rejection of claim".equals(o.getValue().getTypeOfDocument())).findFirst();
+
+        return "Rejected".equals(caseDetails.getState())
```

Comment:
Think there is a constant in et-common for Rejected

Reply:
IntelliJ doesn't seem to pick it up, I did look

Reply:
https://github.com/hmcts/et-common/blob/a7b45b7cabc4ae3c4a8ac955b5a29436677d4c8e/src/main/java/uk/gov/hmcts/ecm/common/model/helper/Constants.java#L40